source/2024.tex

```diff
@@ -0,0 +1,101 @@
+\documentclass[11pt, final]{article}
+\usepackage{tar}
+
+\begin{document}
+
+\reporttitle{Trustees Annual Report\\01 Apr 2023 -- 31 Mar 2024}
+
+\section{Reference and Administration Details}
+\begin{tabular}{l l}
+	\textbf{Charity Name:}      & The UK Python Association \\
+	\textbf{Charity Number:}    & 1173471                   \\
+	\textbf{Principal Address:} & c/o Acconomy              \\
+	                            & Arena Business Centre     \\
+	                            & Holyrood Close            \\
+	                            & Poole BH17 7FJ            \\
+\end{tabular}
+
+\subsection{Trustees}
+\begin{tabular}{l l}
+	Owen Campbell  & Daniele Procida \\
+	Kristian Glass & Rebecca Smith \\
+\end{tabular}
+
+\section{Objectives and Activities}
+The objectives of the UKPA are:
+\begin{enumerate}
+	\item To advance education for the public benefit in the use and understanding of the Python programming language within the UK by:
+	      \begin{enumerate}
+		      \item Organising, presenting and promoting Python related conferences, meetings and events at venues within the UK,
+		      \item Supporting and promoting Python related projects and events organised by UKPA members, and,
+		      \item Supporting UKPA members wishing to attend Python related conferences, meetings and events.
+	      \end{enumerate}
+	\item To advance education for the public benefit in the use and understanding of the Python programming language worldwide by supporting and promoting overseas Python related conferences, meetings and events.
+\end{enumerate}
+
+The primary activity of the UKPA is to organise and run the annual PyCon UK conference.
+
+\section{Structure, Governance and Management}
+
+\subsection{Governing Document}
+The UKPA's governing document is its constitution, the current version of which is available at https://github.com/PyconUK/ukpa-constitution/releases/latest.
+
+\subsection{UKPA Members}
+The UKPA membership is the electoral body to which the trustees are accountable.
+There are two classes of membership:
+
+\begin{enumerate}
+	\item Members: have the full rights and obligations described in the UKPA constitution, including the right to vote on members decisions.
+	\item Associate Members: non-voting class of membership as described in the UKPA constitution.
+\end{enumerate}
+
+\subsection{Trustee Appointment}
+At the AGM held on 24-Sep-2023, Vince Knight retired as a trustee in accordance with section 13.1.1 of the UKPA's constitution.
+
+Becky Smith also retired as a trustee in accordance with sections 13.1.4 and 13.1.5 of the constitution.
+
+Vincent Knight and Becky Smith were elected as trustees for a further term in accordance with section 13.1.4 of the constitution.
+
+On 04-Oct-2023, Vince Knight resigned as a trustee.
```

Comment:
We should probably be consistent everywhere with names; s/Vince/Vincent and s/Becky/Rebecca

Reply:
done